docs/advanced-guides/tour.md

```diff
@@ -0,0 +1,70 @@
+# Tour of Ignite source code
+
+Ignite's source looks a tad intimidating at first, but it's actually quite straightforward once you get some experience moving around in it. Let's take a tour.
+
+(note in any code examples we're using `yarn` ... you can substitute `npm` alternative commands if desired)
+
+## Lerna & "packages"
+
+Ignite is composed of a few "packages" (in the `./packages` folder). You can think of these packages as mini npm packages, all controlled by a nice tool called [Lerna](https://lernajs.io/). This allows us to split up Ignite into separate packages while not having to coordinate releases across multiple repos and npm versions.
+
+All you really need to know about Lerna for these purposes is that you can run this from the main folder:
+
+```sh
+$ yarn run bootstrap
+```
+
+![image](https://cloud.githubusercontent.com/assets/1479215/25318334/12209eaa-2841-11e7-926d-f26113dbefb9.png)
+
+This will automatically run `yarn` in each of the separate packages, which each has its own `node_modules` folder.
+
+## gluegun
+
+`ignite-cli` (described [next](#ignite-cli)) uses another Infinite Red npm package called [gluegun](https://github.com/infinitered/gluegun). Gluegun is sort of like [Yeoman](http://yeoman.io/) in that it gives us a bunch of CLI tools, but it's much lighter and doesn't take over your whole CLI.
+
+Whenever you see `context` in `ignite-cli`, it is a gluegun runtime context. This is a JS object that has a bunch of useful functions and properties attached to it, like `print` and `parameters`.
+
+In addition, Ignite adds some of its own Ignite-specific functions and parameters to `context` ... those are found under `context.ignite`.
+
+If you're curious what's in the context object, check out Gluegun's [nice context docs](https://infinitered.github.io/gluegun/#/context-api.md), or just inspect it with `console.dir(context, {depth: 3, colors true})`.
```

Comment:
Neither did I. I literally looked up `console.dir` to find out what the frig it was, since I'd seen you use it a lot.

https://nodejs.org/api/console.html#console_console_dir_obj_options

Lots of cool stuff.